reference/entry.json

```diff
@@ -2006,6 +2006,259 @@
           "Places"
         ],
         "x-internal": false
+      },
+      "put": {
+        "summary": "",
+        "operationId": "put-places",
```

Comment:
Not sure if we already have a good convention for operation ids (and we don't really use them for anything right now), but maybe `put-place` might be better since you can only put one.